EIPS/eip-5805.md

```diff
@@ -52,51 +52,13 @@ Both timestamps, block numbers, and other possible modes use the same external i
 
 ### Methods
 
-#### clock
+#### [ERC-6372](./eip-6372.md): clock and CLOCK_MODE
 
-This function returns the current timepoint. It could be `block.timestamp`, `block.number` (or any other **non-decreasing** function) depending on the mode the contract is operating on.
+A contract implementing this ERC may optionally implement ERC-6372 (Contract clock) to announce the clock that is used for vote tracking.
```

Comment:
```suggestion
A contract implementing this ERC SHOULD implement ERC-6372 (Contract clock) to announce the clock that is used for vote tracking.
```

Reply:
The rest of the ERC isn't using MUST, SHOULD, etc.

If we want to use those we need a larger review. Let's do that in another PR.

Reply:
Oh, my bad I see that it is using those keywords. Just need to add the notice about them.

Reply:
Yeah, basically you can choose to adopt it or not. I just wanna point out it leaves ambiguity in some cases and could cause challenges for adoption (the contract client side)